### ISSUE TITLE
 'React' has no exported member 'FormHTMLAttributes'

### ISSUE THREAD

**Issue by ThunderDev1** [tkrotoff/react-form-with-constraints#17]
When I install the package, I get the following error

./node_modules/react-form-with-constraints/lib/FormWithConstraints.d.ts:11:57 
    TS2694: Namespace 'React' has no exported member 'FormHTMLAttributes'.

Using react 16.1.1

How can I get this to work ?

**Comment by tkrotoff** [comment]
tl;dr Update `@types/react` to the latest version.

Line 11 https://unpkg.com/react-form-with-constraints@0.6.4/lib/FormWithConstraints.d.ts:
```TypeScript
export interface FormWithConstraintsProps extends React.FormHTMLAttributes<HTMLFormElement> ...
```

Now let's check react/index.d.ts (`@types/react`): https://github.com/DefinitelyTyped/DefinitelyTyped/blame/b138293854757776d2efac17494793cd95300557/types/react/index.d.ts#L2687

`interface FormHTMLAttributes<T>` is defined and has been introduced 4 months ago (Jul 10, 2017), see https://github.com/DefinitelyTyped/DefinitelyTyped/commit/a24aee6125213ba20a5bda15a5d4fce8b60b2f57#diff-96b72df8b13a8a590e4f160cbc51f40cR2576

I guess you are using an old version of `@types/react`. Update to the latest version and see if it works.

**Comment by ThunderDev1** [comment]
Thanks for the fast response, it was as you said, using an old version of @types/react.
